docs/tutorial/prismic-source-graphql-tutorial.md

```diff
@@ -0,0 +1,333 @@
+
+```javascript:title=src/pages/index.js
+import React from "react"
+import { Link, graphql } from "gatsby" //highlight-line
+
+import Layout from "../components/layout"
+import { RichText } from "prismic-reactjs" //highlight-line
+
+//highlight-start
+const export query = graphql`
+{
+  prismic {
+    allBlog_homes {
+      edges {
+        node {
+          headline
+          description
+          image
+        }
+      }
+    }
+    allPosts(sortBy: date_DESC) {
+      edges {
+        node {
+          _meta {
+            id
+            uid
+            type
+          }
+          title
+          date
+        }
+      }
+    }
+  }
+}
+`
+// highlight-end
+```
+    
+
+In order to render this data, replace the `export default IndexPage` line in the original `index.js` file with the following.
+
+```js:title=src/pages/index.js
+export default ({ data }) => {
+  const doc = data.prismic.allBlog_homes.edges.slice(0,1).pop();
+  const posts = data.prismic.allPosts.edges;
+
+  if(!doc) return null;
+
+  return(
+    <Layout>
+      <div>
+        <img src={doc.node.image.url} alt="avatar image" />
+        <h1>{ RichText.asText(doc.node.headline) }</h1>
+        <p>{ RichText.asText(doc.node.description) }</p>
+      </div>
+    </Layout>
+  )
+}
+```
+
+Save the file and check on your site running at `http://localhost:8000`
+
+You can use the helper function RichText to render formatted text (link to resource on rendering) and generally, this is the path you will take to query your Prismic repository and then render it. We can clean this up and include a function that will render the array of queried blog posts.
+
+```js:title=src/pages/index.js
+//highlight-start
+const BlogPosts = ({ posts }) => {
+  if(!posts) return null;
+  return(
+    <div>
+      {posts.map((post) => {
+        return(
+          <div key={post.node.id}>
+            <h2>{ RichText.asText(post.node.title) }</h2>
+            <p>
+              <time>{post.node.date}</time>
+            </p>
+          </div>
+        );
+      })}
+    </div>
+  );
+}
+//highlight-end
+
+export default ({ data }) => {
+  const doc = data.prismic.allBlog_homes.edges.slice(0,1).pop();
+  const posts = data.prismic.allPosts.edges;
+
+  if(!doc) return null;
+
+  return(
+    <Layout>
+      <div>
+        <img src={ doc.node.image.url } alt="avatar" />
+        <h1>{ RichText.asText(doc.node.headline) }</h1>
+        <p>{ RichText.asText(doc.node.description) }</p>
+      </div>
+      <BlogPosts posts={ posts } /> //highlight-line
+    </Layout>
+  )
+}
+```
+
+### Building links to your documents
+
+Now things are really taking shape. We will turn this blog post titles into links by building a link resolver function. (link to resources), which will build the correct route for your posts.
```

Comment:
```suggestion
Now things are really taking shape. We will turn these blog post titles into links by building a link resolver function. (link to resources), which will build the correct route for your posts.
```
Reminder to add a link to resources!

Reply:
This is embarrassing, had left them as placeholder reminders to myself and forgot to put the link before submitting.